cmd/start/main.go

```diff
@@ -432,3 +446,22 @@ func getCommitSHA(dir string) (string, error) {
 	}
 	return strings.TrimSpace(string(content)), nil
 }
+
+func gitLfsInUse(logger logging.LeveledLoggerInterface, dir string) (lfs bool, errBytes []byte, err error) {
+	stdout, stderr, err := command.RunInDir("git", []string{"lfs", "ls-files", "--all"}, dir)
+	if err != nil {
+		return false, stderr, err
```

Comment:
Sorry, I guess my suggestion wasn't really clear. I would do it like this:

```go
return false, fmt.Errorf("cannot list git lfs files: %s (%w)", stderr, err)
```

That way the caller only has to deal with the error, and not both stderr and err. Stderr is the responsibiliy of this function I'd say.